src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeViewNode.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class TreeViewNode : DependencyObject, ICustomPropertyProvider, IStringable
+	{
+		private TreeViewNode _parent;
+		private bool _hasUnrealizedChildren;
+		private object m_itemsSource = null;
+		private ItemsSourceView m_itemsDataSource = null;
+
+		public TreeViewNode()
+		{
+			var collection = new TreeViewNodeVector();
+			collection.SetParent(this);
+			Children = collection;
+			collection.VectorChanged += ChildVectorChanged;
+
+			//this.RegisterDisposablePropertyChangedCallback((s, p, e) => OnPropertyChanged(e));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)